client/platform/desktop/frontend/components/ViewerLoader.vue

```diff
@@ -128,21 +137,21 @@ export default defineComponent({
     </template>
     <template #title-right>
       <RunPipelineMenu
-        :selected-dataset-ids="[id]"
+        :selected-dataset-ids="[modifiedId]"
```

Comment:
Since pipelines can run on multi-camera data, why does this menu take the specific identifier of an individual camera?  Should it take both?

Reply:
The capability to run single cam pipelines on the selected Camera.  Especially stuff that doesn't modify the trackIds like polygon generation pipelines.  There is the warning currently about doing this currently without knowing what pipeline you are running.  I can put a `prop` that is selectedCamera and it just defaults to an ignored value as well.  If you want I can do that.